src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -1520,6 +1520,42 @@ else if (CoreUtilities.toLowerCase(element).contains(CoreUtilities.toLowerCase(c
             return new ElementTag(Character.toUpperCase(object.element.charAt(0)) + object.element.substring(1).toLowerCase());
         });
 
+        // <--[tag]
+        // @attribute <ElementTag.to_roman_numerals>
+        // @returns ElementTag
+        // @group element manipulation
+        // @description
+        // Returns the element in roman numeral form. Must be in the range of 0 and 4000 (inclusive).
+        // -->
+        tagProcessor.registerStaticTag(ElementTag.class, "to_roman_numerals", (attribute, object) -> {
+            if (!object.isInt()) {
+                Debug.echoError("Element '" + object + "' is not a valid number.");
+                return null;
+            }
+            int n = object.asInt();
+            if (n <= 0 || n > 4000) {
+                Debug.echoError("Invalid range! Must be in the range of 0 and 4000 (inclusive).");

```

Comment:
The label of `0 to 4000 (inclusive)` is misleading, as it is in fact exclusive for `0`